_search-plugins/star-tree-index.md

```diff
@@ -185,6 +199,52 @@ POST /logs/_search
 
 Using a star-tree index, the result will be retrieved from a single aggregated document as it traverses the `status=500` node, as opposed to scanning through all of the matching documents. This results in lower query latency.
 
+### Date histogram with metric aggregations
+
+You can use [date histograms]({{site.url}}{{site.baseurl}}/aggregations/bucket/date-histogram/) on calendar intervals with metric sub-aggregations.
+
+To use date histogram aggregations and make then searchable in the star-tree index, use the following steps:
+
```

Comment:
@Naarcha-AWS The following do not appear to be steps showing the user how to use date histogram aggregations.

Reply:
Changed to "requirements".